drivers/dac/dac_handlers.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2020 Libre Solar Technologies GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <drivers/dac.h>
+#include <syscall_handler.h>
+#include <kernel.h>
+
+static inline int z_vrfy_dac_channel_setup(struct device *dev,
```

Comment:
In order for these handler to work, you need to add an entry for `dac_driver_api` to the `subsystems` array in `scripts/gen_kobject_list.py`.  Otherwise no handlers will be generated and compilation of this file will fail.